src/Servers/IIS/IIS/src/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -49,6 +49,10 @@
     <Reference Include="Microsoft.AspNetCore.Http.Features" />
     <Reference Include="Microsoft.AspNetCore.Server.HttpSys" />
     <Reference Include="Microsoft.Extensions.FileProviders.Physical" />
+    <Reference Include="Microsoft.Windows.CsWin32">
```

Comment:
Looks like this project is being built/restored in source-build and this new reference is not able to be resolved.  Can this reference be conditioned out for source-build?  Looking at this more broadly, is this project even necessary for source-build because this is IIS related.